src/EffectsLibrary.js

```diff
@@ -1,218 +1,204 @@
-class EffectsLibrary  
-{
+class EffectsLibrary {
   // $singleton: true, // or is it
   static instance;
 
-    static c_particleLimit = 5000;
+  static c_particleLimit = 5000;
 
-    static globalPercentMin        = 0;
-    static globalPercentMax        = 20.0;
-    static globalPercentSteps      = 100.0;
+  static globalPercentMin = 0;
```

Comment:
FYI anything declared w/ `static` can't be called with `this.`

Reply:
No need to change anything here?